src/main/java/org/folio/service/export/LocalFileSystemExportService.java

```diff
@@ -94,6 +101,36 @@ public void exportSrsRecord(Pair<List<String>, Integer> marcToExport, ExportPayl
     }
   }
 
+  private void handleControlCharacters(String jsonRecord, String jobExecutionId, OkapiConnectionParams params) {
+    final String controlCharacterMarker = UUID.randomUUID().toString();
+    ErrorCode errorCode = ERROR_MARC_RECORD_CONTAINS_CONTROL_CHARACTERS;
+    // Replace all control characters with markers, otherwise JsonObject cannot be instantiated.
+    jsonRecord = jsonRecord.replaceAll(CONTROL_CHARACTERS_REPLACE_PATTERN, controlCharacterMarker);
```

Comment:
Is it possible to use some exists escape utils?

Reply:
I tried, but utils I could find are not working as expected.